cni/network/plugin/main.go

```diff
@@ -90,8 +90,6 @@ func rootExecute() error {
 	cniCmd := os.Getenv(cni.Cmd)
 
 	if cniCmd != cni.CmdVersion {
-		logger.Info("Environment variable set", zap.String("CNI_COMMAND", cniCmd))
```

Comment:
This is very useful log as I tend use it an identifier that a certain flow has started. Opinions @tamilmani1989 @behzad-mir?

Reply:
yes lets not remove this